src/test/shell/integration/nonincremental_builds_test.sh

```diff
@@ -64,6 +64,23 @@ if "$is_windows"; then
   export MSYS2_ARG_CONV_EXCL="*"
 fi
 
+if "$is_windows"; then
+  EXE_EXT=".exe"
+else
+  EXE_EXT=""
+fi
+
+ javabase="$1"
+if [[ $javabase = /* || $javabase =~ [A-Za-z]:[/\\] ]]; then
```

Comment:
I just realized you don't need this check -- also in your other PRs. `rlocation` checks if the path is absolute.

You can simplify this logic (lines 74..81) to:
```
if [[ $javabase = external/* ]]; then
  javabase=${javabase#external/}
fi
jmaptool="$(rlocation "${javabase}/bin/jmap${EXE_EXT}")"
```